docs/source/api_reference/models.rst

```diff
@@ -393,6 +393,11 @@ Note that this is an alternative implementation for MPNN and currently you can o
 .. autoclass:: deepchem.models.torch_models.MPNNModel
   :members:
 
+InfoGraphModel
```

Comment:
All the layers should be in the docs. We're trying to build up and document DeepChem's collection of layers so users can start to reuse layers